OpenRA.Mods.AS/Activities/ChronoResourceTeleport.cs

```diff
@@ -42,10 +45,14 @@ public override bool Tick(Actor self)
 			foreach (var ost in self.TraitsImplementing<IOnSuccessfulTeleportRA2>())
 				ost.OnSuccessfulTeleport(info.TeleportType, sourcepos, self.CenterPosition);
 
+			var facing = self.TraitOrDefault<IFacing>();
```

Comment:
Upon testing, we have to set facings because the turning of MoveOnToTarget is not working.